doc/source/io.rst

```diff
@@ -4806,6 +4806,55 @@ default ``Text`` type for string columns:
     Because of this, reading the database table back in does **not** generate
     a categorical.
 
+.. _io.sql.method:
+
+Insertion Method
+++++++++++++++++
+
+.. versionadded:: 0.24.0
+
+The parameter ``method`` controls the SQL insertion clause used.
+Possible values are:
+
+- ``'default'``: Uses standard SQL ``INSERT`` clause (one per row).
```

Comment:
The "one per row", is that correct? 

Because as far as I understand, the default behaviour will be to do a single insert statement with multiple rows (a "executemany"), where the number of rows is determined by the chunksize (and which by default is all rows)

Reply:
> The "one per row", is that correct?

Maybe that is why everybody is complaining that it is so slow :grin: